test/PySrc/tests/test_live_image.py

```diff
@@ -236,6 +236,18 @@ def test_differ_compare():
     assert diff_pixel2 == expected_diff
     assert differ.diff_count == 1
 
+@pytest.mark.skipif(Image is None, reason='Pillow not installed.')
+def test_compare_pixel_alpha_channel_missing():
+
+    image1 = LivePillowImage(Image.new('RGB', (100, 200)))
+    image2 = LivePillowImage(Image.new('RGB', (100, 200)))
+
+    differ = LiveImageDiffer()
+    try:
+        differ.compare(image1, image2)
+    except Exception as e:
```

Comment:
Thanks for adding the test, @tj-26. One suggestion: you don't need to catch the exception and assert false. Just let the exception fly!

Pytest actually distinguishes between failures (assertions failed) and errors (exceptions raised) in its summary report.

Reply:
Hey @donkirkby thanks for that suggestion. Ill make the change and ask for another review. Thanks